beginning.md

```diff
@@ -256,30 +260,29 @@ TypeError: cannot concatenate 'str' and 'int' objects
 ```
 
 ## Writting a program in file
-* All programs are written with *<filename>.py*
-* Open a file named *first_program.py* and write the following
-  code.
+* All programs are written with `<filename>.py`.
+* Open a file named `first_program.py` and write the following code.
```

Comment:
in any editor like (vim, gedit, code, atom or notepad++)

Reply:
I would like to add more generic statement like 
I don't like to use specific editor name.